contracts/InterestRateModels/InterestRateModel.sol

```diff
@@ -31,4 +31,13 @@ contract InterestRateModel {
         uint reserves,
         uint reserveFactorMantissa
     ) external view returns (uint);
```

Comment:
Should we remove `getSupplyRate` from InterestRateModel?

Reply:
Fixed in [commit](https://github.com/VenusProtocol/venus-protocol/pull/210/commits/6b9a0d6e81b2c8e15489d629723beb649c386738)